external/CMakeLists.txt

```diff
@@ -304,4 +304,42 @@ set_property(TARGET xxhash PROPERTY FOLDER externals)
 # Soundtouch audio processing library for timing, pitch and playback rate manipulation
 add_subdirectory(soundtouch)
 set_property(TARGET SoundTouch PROPERTY FOLDER externals)
-set_property(TARGET soundstretch PROPERTY FOLDER externals)
\ No newline at end of file
+set_property(TARGET soundstretch PROPERTY FOLDER externals)
+
+# Tracy
+option(TRACY_ENABLE "Enable profiling of the emulator using Tracy" ON)
```

Comment:
CI builds will never have Tracy as CMake already disables Tracy on `Release` and `MinSizeRel` builds by not defining the `TRACY_ENABLE` pre-processor macro. Also I might rename that CMake option since it's just to control whether the `module` (`vita3k/module`) CMake target links against Tracy or not so that HLE modules can be compiled without requiring the parent project to have Tracy support in their build system, like in vita2hos.

Reply:
gotcha